index.html

```diff
@@ -2,13 +2,95 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title> Taher Haggui </title>
-        <link rel="stylesheet" type="text/css" href="styles/template2.css">
+        <title> Jamstack </title>
+        <link rel="stylesheet" type="text/css" href="styles/home_style.css">
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap" rel="stylesheet">
+        <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
         <meta name="viewport" content="width=device-width, initial-scale=1">
     </head>
     <body>
+        <section class="home-page">
+          <header>
+            <nav class="desktop">
+                <a href="#home" class="logo"> <img src="statics/images/logo.svg" alt="logo"></a>
+                <ul class="menu">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Program</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#">News</a></li>
+                </ul>
+                <p>JD Campaign</p>
+            </nav>
+            <nav class="mobile">
```

Comment:
- Provide margins between each ``li`` in your design:
<img width="564" alt="Screen Shot 2021-10-14 at 7 21 08" src="https://user-images.githubusercontent.com/22258274/137316281-94fbfe9c-b376-4025-a0f6-932ef46adec1.png">